solvers/ConfigurationSpaceMotionPlanningTree.m

```diff
@@ -0,0 +1,131 @@
+classdef ConfigurationSpaceMotionPlanningTree < CartesianMotionPlanningTree
```

Comment:
Agreed. I'll rename this class to `JointSpaceMotionPlanningTree`, since we usually treat the joint space as a Cartesian space.